### PR TITLE
Update Magento_CacheInvalidate ReadMe

### PR DESCRIPTION
**Pull request by korovitskyi**
### Description (*)

Update README.md file for Magento_CacheInvalidate module.

### Fixed Issues (if relevant)

1. magento/devdocs magento/devdocs#4451: README content for modules - work in M2 Code

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/CacheInvalidate/README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
-The CacheInvalidate module is used to invalidate the Varnish cache if it is configured.
-It listens for events that request the cache to be flushed or cause the cache to be invalid, then sends Varnish a purge request using cURL.
\ No newline at end of file
+# Magento_CacheInvalidate module
+
+The **Magento_CacheInvalidate** module is used to invalidate the Varnish cache if it is configured.
+
+The **Magento_CacheInvalidate** module listens for events that request the cache to be flushed or cause the cache to be invalid, then sends Varnish a purge request using cURL.
+
+## Extensibility
```
Comment (orlangur) [inline app/code/Magento/CacheInvalidate/README.md]: Is it really applicable here? I don't see a lot of classes to customize.

**Review by orlangur — CHANGES_REQUESTED** [review]

File: app/code/Magento/CacheInvalidate/README.md (hunk 1 above)
Comment (korovitskyi) [reply app/code/Magento/CacheInvalidate/README.md]: Done

**Comment by magento-engcom-team** [comment]
Hi @korovitskyi
Thank you for your contribution. This pull request has been combined (merged) with https://github.com/magento/magento2/pull/31304 to optimize pull requests queue on magento/magento2 repository
Closing this pull request.

**Comment by magento-engcom-team** [comment]
Hi @korovitskyi
Thank you for your contribution. This pull request has been combined (merged) with https://github.com/magento/magento2/pull/31304 to optimize pull requests queue on magento/magento2 repository
Closing this pull request.

**Pull request closed without merging**